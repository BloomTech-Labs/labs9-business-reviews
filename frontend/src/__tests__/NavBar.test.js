import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import NavBarComponent from '../components/NavBar';

describe('NavBarComponent />', () => {
  it('renders', () => {
    shallow(<NavBarComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<NavBarComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
