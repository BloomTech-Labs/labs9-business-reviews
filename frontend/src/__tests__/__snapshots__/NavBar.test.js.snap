// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBarComponent /> matches snapshot 1`] = `
<styled.div>
  <div
    className="logo_container"
  >
    <a
      href="/"
    >
      <img
        alt="logo"
        className="logo"
        src="logo.png"
      />
    </a>
  </div>
  <div
    className="nav_links"
  >
    <a
      href="/#Businesses"
    >
      businesses
    </a>
    <a
      href="/#Reviewers"
    >
      reviewers
    </a>
    <a
      href="/billing"
    >
      go pro âœ”
    </a>
  </div>
  <div
    className="menu__user"
  >
    <Link
      className="menu__user--text"
      replace={false}
      to="/login"
    >
      sign in
    </Link>
    <Link
      className="menu__user--text"
      replace={false}
      to="/register"
    >
      sign up
    </Link>
  </div>
</styled.div>
`;
